src/write.jl

```diff
@@ -2,28 +2,31 @@
 ##### utilities
 #####
 
-_edf_write(io::IO, value::Union{String,Char}) = Base.write(io, value)
-_edf_write(io::IO, date::Date) = Base.write(io, uppercase(Dates.format(date, dateformat"dd-u-yyyy")))
-_edf_write(io::IO, date::DateTime) = Base.write(io, Dates.format(date, dateformat"dd\.mm\.yyHH\.MM\.SS"))
-_edf_write(io::IO, x::Integer) = Base.write(io, string(x))
-_edf_write(io::IO, x::AbstractFloat) = Base.write(io, string(isinteger(x) ? trunc(Int, x) : x))
+_edf_repr(value::Union{String,Char}) = value
+_edf_repr(date::Date) = uppercase(Dates.format(date, dateformat"dd-u-yyyy"))
+_edf_repr(date::DateTime) = Dates.format(date, dateformat"dd\.mm\.yyHH\.MM\.SS")
+_edf_repr(x::Integer) = string(x)
+_edf_repr(x::AbstractFloat) = string(isinteger(x) ? trunc(Int, x) : x)
```

Comment:
Can we alter this to insure it will not introduce scientific notation into fields where it's disallowed (e.g. annotation onsets) and/or when it does introduce such notation, that `E` is used instead of `e`?

Reply:
ok sweet on it